Ankit barnawal08 patch 1

AnkitBarnawal08 opened the pull request:
This pull request contain my design assignment of UI/UX. i have created a user friendly and visually appealing interface, focusing on usability, accessibility and responsiveness.
you can view the complete design in Figma here: https://www.figma.com/design/OR15S4CZmBqpcIGhEPLgGu/Acehub.Ai?node-id=0-1&t=h0af4bgRWKjEmFbH-1
Looking forward for your feedback. Thank You!

AnkitBarnawal08 committed:
Create Ankit_figma_link

This Webpage serves as a portfolio to showcase my design .

AnkitBarnawal08 committed:
Merge pull request #1 from AnkitBarnawal08/AnkitBarnawal_UIUX

Create Ankit_figma_link

AnkitBarnawal08 committed:
Update Ankit_figma_link

nayanbamnote commented:
Request access is needed to view the design. Please make it public or accept the request.

AnkitBarnawal08 committed:
Update Ankit_figma_link

This pull request contain my design assignment of UI/UX. i have created a user friendly and visually appealing interface, focusing on usability, accessibility and responsiveness.
you can view the complete design in Figma here: https://www.figma.com/design/OR15S4CZmBqpcIGhEPLgGu/Acehub.Ai?node-id=0-1&t=YAXKztkLxsma55Wt-1
Looking forward for your feedback. Thank You!

AnkitBarnawal08 committed:
Update Ankit_figma_link

This pull request contain my design assignment of UI/UX. i have created a user friendly and visually appealing interface, focusing on usability, accessibility and responsiveness.
you can view the complete design in Figma here: https://www.figma.com/design/OR15S4CZmBqpcIGhEPLgGu/Acehub.Ai?node-id=0-1&t=YAXKztkLxsma55Wt-1
Looking forward for your feedback. Thank You!

AnkitBarnawal08 commented:
Sorry sir for inconvenience caused. But sir I have changed it to view all.

Regards
Ankit Barnawal

On Tue, 4 Mar, 2025, 8:05 pm nayanbamnote, ***@***.***> wrote:

> Request access is needed to view the design. Please make it public or
> accept the request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/neugence/designAssignment/pull/13#issuecomment-2697848701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKPQ2JRDEU4QW5ETKZP6QLT2SW24ZAVCNFSM6AAAAABYHTK4K6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJXHA2DQNZQGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: nayanbamnote]*nayanbamnote* left a comment
> (neugence/designAssignment#13)
> <https://github.com/neugence/designAssignment/pull/13#issuecomment-2697848701>
>
> Request access is needed to view the design. Please make it public or
> accept the request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/neugence/designAssignment/pull/13#issuecomment-2697848701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKPQ2JRDEU4QW5ETKZP6QLT2SW24ZAVCNFSM6AAAAABYHTK4K6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJXHA2DQNZQGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

neugence commented:
Duplicate submission ?

AnkitBarnawal08 commented:
Yes, as the first link I had send was had do not any access to view so I
had submitted second time

On Sat, 8 Mar, 2025, 6:03 am Neugence, ***@***.***> wrote:

> Duplicate submission ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/neugence/designAssignment/pull/13#issuecomment-2707784507>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKPQ2JSI47ZTUGAS6P2RIRL2TI3FFAVCNFSM6AAAAABYHTK4K6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBXG44DINJQG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: neugence]*neugence* left a comment (neugence/designAssignment#13)
> <https://github.com/neugence/designAssignment/pull/13#issuecomment-2707784507>
>
> Duplicate submission ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/neugence/designAssignment/pull/13#issuecomment-2707784507>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKPQ2JSI47ZTUGAS6P2RIRL2TI3FFAVCNFSM6AAAAABYHTK4K6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBXG44DINJQG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
